examples/genai/src/payload.config.ts

```diff
@@ -0,0 +1,38 @@
+import path from 'path'
+
+import { payloadCloud } from '@payloadcms/plugin-cloud'
+import { mongooseAdapter } from '@payloadcms/db-mongodb'
+import { webpackBundler } from '@payloadcms/bundler-webpack'
+import { slateEditor } from '@payloadcms/richtext-slate'
+import { buildConfig } from 'payload/config'
+
+import Users from './collections/Users'
+import { Prompt } from './collections/Prompt'
+import { Transformer } from './collections/Transformer'
+import { Embedding } from './collections/Embedding'
+import { Media } from './collections/Media'
+import { KnowledgeGraph } from './collections/KnowledgeGraph'
+import { Resource } from './collections/Resource'
+
+export default buildConfig({
+  admin: {
+    user: Users.slug,
+    bundler: webpackBundler(),
+    webpack: (config) => {
+      // TODO: How do I support 'fs' module?
```

Comment:
I'm having this issue: @jmikrut 
[unable-to-write-files-using-fs-nodejs](https://payloadcms.com/community-help/discord/unable-to-write-files-using-fs-nodejs)

I am using `fs` in the `Media` collection virtual field, `genai` [here](https://github.com/payloadcms/payload/pull/4930/files#diff-d0cfce4c608f49c0efe6a42a68a11ce44149a8dd8a607c7059624e659c19f813)